addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/internal/api/ApiAccess.java

```diff
@@ -19,10 +20,12 @@
 
 public class ApiAccess {
     private final Logger logger = LoggerFactory.getLogger(ApiAccess.class);
+    private final HttpClient httpClient;
+
+    // TODO remove static?
     private static Authentication authenticationData;
```

Comment:
Yeah but I think that won't work now. The `ApiAccess` is shared left, right and center. I guess the remark is: make it so that it shouldn't be static. Something to work on for me...